build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.2.2'
```

Comment:
FWIW, I'd be 👍 on going beta-bleeding edge. Since we're the maintainers primarily responsible for upkeep, it makes sense to keep it aligned to our prefs/normal environment.

Also since the build tools don't directly affect the lib we publish to jCenter (eg, we don't have to republish the lib when build tools update), we can keep master up to date easily.

Up to you!